docs/guides/user-task-listener.md

```diff
@@ -9,11 +9,15 @@ description: "In Camunda 8, user task listeners allow you to run custom logic wh
 
 import SaasPrereqs from './react-components/\_saas-prerequisites.md'
 
-In Camunda 8, **user task listeners** allow you to run custom logic in response to changes to a user task, for example when it's creating, assigning, or completing.
+In Camunda 8, **user task listeners** allow you to run custom logic in response to changes to a user task—for example, when it's created, assigned, or completed.
```

Comment:
> creating, assigning, or completing.
> created, assigned, or completed.

@christinaausley I understand why you'd want this change for correctness of grammar, but the events are called `creating`, `assigning`, and `completing`. Can we somehow find a different way to describe it that aligns with the terminology used in the product.